Content.Shared/Silicons/Borgs/Components/BorgChassisComponent.cs

```diff
@@ -13,80 +14,132 @@ namespace Content.Shared.Silicons.Borgs.Components;
 /// "brain", legs, modules, and battery. Essentially the master component
 /// for borg logic.
 /// </summary>
-[RegisterComponent, NetworkedComponent, Access(typeof(SharedBorgSystem))]
-[AutoGenerateComponentState, AutoGenerateComponentPause]
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState, AutoGenerateComponentPause]
+[Access(typeof(SharedBorgSystem))]
 public sealed partial class BorgChassisComponent : Component
 {
+    /// <summary>
+    /// Is this borg currently activated?
+    /// If activated the borg
+    /// - has door access,
+    /// - can use modules and
+    /// - has full movement speed.
+    /// To be activated the borg
+    /// - needs to have a player controlling it (a mind),
+    /// - needs enough charge in its power cell and
+    /// - needs to be alive (not crit or dead).
+    /// </summary>
+    /// <remarks>
+    /// Don't try to use ItemToggle for this.
+    /// It used that before and it had a ton of conflicts with other item toggling behavior from the billion components that use it somehow.
+    /// </remarks>
+    [DataField, AutoNetworkedField]
+    public bool Active;
+
+    /// <summary>
+    /// The sound to play when the borg activates.
+    /// </summary>
+    /// <remarks>
+    /// The same as the flashlight. This playing used to be a bug, but the sound is nostalgic at this point, so I'm keeping it.
+    /// </remarks>
+    [DataField]
+    public SoundSpecifier ActivateSound = new SoundPathSpecifier("/Audio/Items/flashlight_on.ogg");
+
+    /// <summary>
+    /// The sound to play when the borg deactivates.
+    /// </summary>
+    [DataField]
+    public SoundSpecifier DeactivateSound = new SoundPathSpecifier("/Audio/Items/flashlight_off.ogg");
```

Comment:
```suggestion
    [DataField, AutoNetworkedField]
    public SoundSpecifier ActivateSound = new SoundPathSpecifier("/Audio/Items/flashlight_on.ogg");

    /// <summary>
    /// The sound to play when the borg deactivates.
    /// </summary>
    [DataField, AutoNetworkedField]
    public SoundSpecifier DeactivateSound = new SoundPathSpecifier("/Audio/Items/flashlight_off.ogg");
```

Feel free to ignore, i just like when sounds are networked for admeme funsies.